manifests/0000_30_cluster-api_00_credentials-request.yaml

```
@@ -70,14 +70,77 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     release.openshift.io/feature-set: "TechPreviewNoUpgrade"
 spec:
+  serviceAccountNames:
+    - cluster-capi-operator
   secretRef:
     name: capz-manager-bootstrap-credentials
     namespace: openshift-cluster-api
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: AzureProviderSpec
-    roleBindings:
-    - role: Contributor
+    permissions:
+    - Microsoft.ApiManagement/service/groups/delete
+    - Microsoft.ApiManagement/service/groups/read
+    - Microsoft.ApiManagement/service/groups/write
+    - Microsoft.ApiManagement/service/workspaces/tags/read
+    - Microsoft.ApiManagement/service/workspaces/tags/write
+    - Microsoft.Authorization/roleAssignments/read
+    - Microsoft.Authorization/roleAssignments/write
+    - Microsoft.Compute/availabilitySets/delete
+    - Microsoft.Compute.availabilitySets/write
```

I think you have a typo here, causing a failure.
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-azure-modern/1700029488332541952

```suggestion
    - Microsoft.Compute/availabilitySets/write
```

---

Fixed, thanks!